netlify/functions/manage-findings.ts

```diff
@@ -135,22 +94,36 @@ async function getFindings(
     teams: {},
   };
 
-  const promises: Promise<void | void[]>[] = [];
+  const allFindings = await getAllFindings(
+    octokit,
+    repoName,
+    process.env.GITHUB_CONTEST_REPO_OWNER!
+  );
+
+  const wardenAndTeamFindingRequests: Promise<void | void[]>[] = [];
   if (includeTeams) {
     const teamNames = await getUserTeams(username);
-    promises.push(
-      getTeamsFindings(teamNames, octokit, contest, wardenFindingsForContest)
-    );
+    for (const team of teamNames) {
+      wardenAndTeamFindingRequests.push(
```

Comment:
if there's a risk of "Promise.all traffic jam" here you could now use https://github.com/code-423n4/api.code4rena.com/blob/main/Utils/utils.ts#L353 as documented https://gist.github.com/natevw/79e04dba0cda6c87e19eb63a6a4986ba

Reply:
Yep, fair point. However, it's extremely rare for a warden to be on more than 1 team, and I don't know of any that are on more than 2, so this will be at most 3 promises - at least for now.